actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -146,6 +148,12 @@ def _filtered_referrer # :nodoc:
       #   link_to nil, "http://example.com"
       #   # => <a href="http://www.example.com">http://www.example.com</a>
       #
+      # More concise yet, when +name+ is an ActiveRecord model that defines a
+      # +to_s+ method returning a default value or a model instance attribute
```

Comment:
@zzak I changed this from: 
> +to_s+ method returning a name attribute or a default value
 
To this: 
> +to_s+ method returning a default value or a model instance attribute

I feel like the mention of a "name attribute" could have been misunderstood as the `+name+` argument to `link_to` when really I meant a completely different model instance attribute called `name`. That was too confusing, the implementation of whatever custom `#to_s` is irrelevant.